docs/storagepolicies.md

```diff
@@ -144,19 +169,170 @@ metadata:
+metadata:
+  annotations:
+    cas.openebs.io/config: |
+      - name: VolumeControllerImage
+        value: quay.io/openebs/cstor-volume-mgmt:v0.7.x-ci
+    openebs.io/cas-type: cstor
+```
+
+### Volume Target Image Policy
+
+You can specify the cStor Target Image using the *value* for *VolumeTargetImage* property. This will help to choose the cStor istgt target image.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    cas.openebs.io/config: |
+      - name: VolumeTargetImage
+        value:quay.io/openebs/cstor-istgt:v0.7.x-ci
+    openebs.io/cas-type: cstor
+```
+
+### Storage Pool Claim Policy
+
+You can specify the cStor Pool Claim name using the *value* for *StoragePoolClaim* property. This will help to choose cStor storage pool name where OpenEBS volume will create. Following is the default StorageClass template where cStor volume will create on default cStor Sparse Pool.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    cas.openebs.io/config: |
+      - name: StoragePoolClaim
+        value: "cstor-sparse-pool"
+    openebs.io/cas-type: cstor
+```
+
+### Volume Monitor Policy
+
+You can specify the cStor volume monitor feature which can be set using *value* for *VolumeMonitor* property.  By default, volume monitor is enabled.
+
+**Note:**
+
+cStor Volume Monitor is a docker image. Following is a sample that makes use of Volume monitor setting policy.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    cas.openebs.io/config: |
+      - enabled: "true"
+        name: VolumeMonitor
+    openebs.io/cas-type: cstor
+```
+
+### Volume Monitoring Image Policy
+
+You can specify the monitoring image policy for a particular volume using *value* for *VolumeMonitorImage* property. The following sample storage class uses the Volume Monitor Image policy.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    cas.openebs.io/config: |
+      - name: VolumeMonitorImage
+        value: quay.io/openebs/m-exporter:v0.7.x-ci
+    openebs.io/cas-type: cstor
+```
+
+### Volume File System Type Policy
+
+You can specify the file system type for the cStor volume where application will consue the storage using *value* for *FSType*. The following is the sample storage class. Currently OpenEBS support ext4 as the default file system and it also support XFS.
+
+```
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  annotations:
+    cas.openebs.io/config: |
+      - name: FSType
+        value: ext4
+    openebs.io/cas-type: cstor
+```
+
+### Volume Space Reclaim Policy
+
+Support for a storage policy that can disable the cStor Volume Space reclaim.You can specify the cStor volume space reclaim feature setting using the *value* for *RetainReplicaData* property. In the following example, the cStor volume space reclaim feature is enabled as true. Hence, retain the volume data post PVC deletion.
```

Comment:
OpenEBS supports a storage policy that can disable...reclaim. (space)
...for the *RetainREplicaData* property.
Hence, it retains the volume data after deleting the PVC.